### ISSUE TITLE
CloudQueue AddMessage* is not guarding against lower bound values correctly

### ISSUE THREAD

**Issue by SeanFeldman** [Azure/azure-storage-net#540]
The client is filtering any `timeToLive` lower than `TimeSpan.Zero`, so anything below 1 second and above 0 [passes the check](https://github.com/Azure/azure-storage-net/blob/7be6a470cf07c0db3a97f214d139bfe9cf5f623b/Lib/WindowsRuntime/Queue/CloudQueue.cs#L1113). But right after that, an [inaccurate conversion is made](https://github.com/Azure/azure-storage-net/blob/7be6a470cf07c0db3a97f214d139bfe9cf5f623b/Lib/WindowsRuntime/Queue/CloudQueue.cs#L1114), causing a value that would round up to a second become a zero and server throwing an exception.

Consider changing how the code check the lower boundary to something like:
```c#
var roundedUp = Convert.ToInt64(Math.Ceiling(timeToLive.Value.TotalSeconds));
CommonUtility.AssertInBounds(//using rounded up value);
```

**Comment by erezvani1529** [comment]
Thanks @SeanFeldman! 

We will make a fix for this. If you are interested we will be happy to take your PR as well.

Thanks,
Elham

**Comment by SeanFeldman** [comment]
I'd love to, but...

Have looked into [contributing.md](https://github.com/Azure/azure-storage-net/blob/master/CONTRIBUTING.md) and you're asking for the following:

1. Visual Studio 2013 and Visual Studio 2015 are required - sorry, this is way to much to expect. My environment only has 2017.
1.  Fiddler 2 - really?
1. You're asking to install FxCop - just no. With Roslyn analyzers that can be part of projects there shouldn't be a requirement to install FxCop.

Given these requirements, I am not able to submit a PR. Sorry.

**Comment by SeanFeldman** [comment]
@seanmcc-msft could you please to the PR that fixes this issue? For an outsider it looks like it will be fixed in the milestone 9.4, but no PR linked to this issues/work done. Thank you.

**Comment by seanmcc-msft** [comment]
Here is the commit:

https://github.com/Azure/azure-storage-net/commit/c5e2574695003f185653929f504398e42a95a483
